sync-orders-to-bigQuery/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "moltin-google-bigquery",
+  "version": "0.1.0",
+  "description": "send order and order items to Google BigQuery",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Matthew Foyle",
+  "license": "MIT",
+  "devDependencies": {
+    "@moltin/eslint-config": "^1.0.0",
+    "eslint": "^5.16.0",
+    "serverless-google-cloudfunctions": "*"
+  },
+  "dependencies": {
+    "@google-cloud/bigquery": "^4.1.2",
+    "@google-cloud/kms": "^1.1.0",
+    "@moltin/request": "^2.0.1",
+    "fs": "^0.0.1-security",
```

Comment:
What's the reason behind this one?

Reply:
I'm seeing four dependancies linked to your comment, was your question related to a specific one?